### PR TITLE
Updated yotta Docker instructions since OSS

### PR DESCRIPTION
**Pull request by thegecko**
@iriark01

### CODE REVIEW

File: Docs/docker_install.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Docker containers let you run an application such as yotta, with all of its dependencies (like Python), without installing anything directly on your machine. This protects you from version conflicts and other installation problems. 
 
-We’ve built a yotta Docker container - a yocker - and a bash script (``yotta.sh``). The script allows you to use the yotta Docker container to mimic a yotta installation on your system; yotta will be fully functional.
+We’ve built a yotta Docker container and a bash script (``yotta.sh``) which allows you to use the yotta Docker container to mimic a yotta installation on your system; yotta will be fully functional.
```
Comment (iriark01) [inline Docs/docker_install.md]: , which

File: Docs/docker_install.md [hunk 2]
```diff
@@ -46,5 +46,9 @@ yotta target frdm-k64f-gcc
 yotta build
 ```
 
+### Open Source
 
+The yotta Docker image is open source, feel free to fork it:
```
Comment (iriark01) [inline Docs/docker_install.md]: ; feel 
or
, so feel

File: Docs/docker_install.md [hunk 3]
```diff
@@ -46,5 +46,9 @@ yotta target frdm-k64f-gcc
...
+
+https://github.com/ARMmbed/yotta-docker
```
Comment (iriark01) [inline Docs/docker_install.md]: [feel free to fork it](https://github.com/ARMmbed/yotta-docker).

**Comment by iriark01** [comment]
Added comments

**Comment by thegecko** [comment]
Change as you see fit

File: Docs/docker_install.md (hunk 3 above)
Comment (thegecko) [inline Docs/docker_install.md]: IMO You shouldn't really hide links behind 'click here' type text

**Comment by thegecko** [comment]
Updated

**Pull request merged**